docs/source/4git_users.rst

```diff
@@ -0,0 +1,103 @@
+DataLad for git/git-annex users
+*******************************
+
+Overview
+========
+
+At its core, DataLad can be considered a thin layer on top of ``git`` and
+``git-annex`` that tries to unify and simplify working with repositories.
+DataLad does not store additional state information or use a content store
+outside of ``.git/objects`` and ``.git/annex/objects``.  You can work with
+DataLad datasets as with any other git/git-annex repository, or you can use
+DataLad to work with regular git/git-annex repositories.  DataLad extensively
+relies on git's builtin ``submodule`` mechanism to manage a hierarchy of
+repositories (:term:`superdataset` in DataLad terms), so you could even use
+``git submodule`` directly.
```

Comment:
I think somewhere we should explain our use of non-absorbed submodules because anyone familiar with submodules outside of DataLad might be surprised by this, but I doubt this is the place to do that.